### ISSUE TITLE
Issues with DK Bongos and Nintendont

### ISSUE THREAD

**Issue by el-quinto** [FIX94/Nintendont#780]
If I launch Donkey Konga 1 or 2 with the DK Bongos plugged in, the menus are stuck continuously scrolling. I have to unplug/replug the controller to get it to register in the game as DK Bongos and stop scrolling. Even then, the "back" function on the bongos (press and hold both pads) is non-functional.

I'm using the latest revisions of nintendont/usb loader gx, I don't have any of these issues if I launch the Donkey Konga games with DIOS MIOS [lite]/usb loader gx.

**Comment by chipsnblip** [comment]
hello, try setting _Native Control_ to **On** in the game's settings

**Comment by el-quinto** [comment]
I just tried with Native Control to On and the DK Bongo issues persist.

**Comment by el-quinto** [comment]
Nevermind my last comment. I double-checked everything and it does indeed seem to work normally with native control  on. My only problem now is that with native control on I can't use the button combo to exit the game, I have to do a full manual shutdown.

**Comment by chipsnblip** [comment]
yes, i believe that is a side-effect of using native controls.  more info here: https://gbatemp.net/threads/nintendont.349258/#ControllersNat

**Comment by el-quinto** [comment]
If I upgraded to a Wii U and got the gc controller adapter, would I be stuck having these issues with the bongos because Native Control requires a Wii with gamecube ports?

**Comment by carnage702** [comment]
> If I upgraded to a Wii U and got the gc controller adapter, would I be stuck having these issues with the bongos because Native Control requires a Wii with gamecube ports?

Probably yes

**Comment by el-quinto** [comment]
I found a thread where Howard said a fix for these issues with the DK Bongos was committed in v2.273. https://gbatemp.net/threads/so-is-it-possible-to-get-the-dk-bongos-working-on-wii-u-with-nintendont-with-the-gc-adapter.375432/page-3

I'm definitely using the latest boot.dol, but for some reason I still have the issues.

**Comment by carnage702** [comment]
> 
> 
> I found a thread where Howard said a fix for these issues with the DK Bongos was committed in v2.273. https://gbatemp.net/threads/so-is-it-possible-to-get-the-dk-bongos-working-on-wii-u-with-nintendont-with-the-gc-adapter.375432/page-3
> 
> I'm definitely using the latest boot.dol, but for some reason I still have the issues.

well 273 says improved gamecube adapter for wii u bongo support so im guessing its just for the adapter and not for the wii native gc ports, the only way to be fure would be to purchase one officialy if possible and connect your bongos there,  keep in mind wiiu gc adapter works on wii or wiiu with nintendont so you dont exactly need a wiiu to use them.

**Comment by geogolem** [comment]
i did notice bongos code for the wii u adapter when i was mucking in the code the other day. I dont have any bongos so i cant test it out, but there is code there, and if the bongos behave incorrectly it should be fixable IMO..
